coderd/database/queries/templates.sql

```diff
@@ -115,8 +115,10 @@ UPDATE
 	templates
 SET
 	updated_at = $2,
-	default_ttl = $3,
-	max_ttl = $4
+	allow_user_auto_start = $3,
```

Comment:
Oh, is it safe and backward-compatible to change the order of items here?

Reply:
It shouldn't matter since the generated Golang query uses named parameters in a struct, not ordered parameters in a function.